logs/app.log

```diff
@@ -0,0 +1,9 @@
+2023-09-21 17:28:43,423 INFO mate.academy.service.AuthenticationServiceImpl:12 login method was called. Params: login = bob.
```

Comment:
this folder should not be on GitHub. Let's edit the .gitignore file. And let's write more informative messages for the commits

Reply:
I can't delete something from gitHub, i just add folder in .gitignore, that all i can do

Reply:
@KDanil2000 try to remove that folder locally

then run your code so that it appears again

and then check how your IDE highlights it - should be something like that:
![image](https://github.com/mate-academy/jv-logger/assets/85908570/356ac528-258d-4bdf-ae6d-91b8bb45c23d)
as you can see `HELP.md` is a bit more of a grey color, which means it is ignored by git